react/homework.md

```diff
@@ -0,0 +1,127 @@
+# React homework
```

Comment:
Could you combine the sub-homework into one big homework.

Reply:
done